src/test/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtilsUnitTest.java

```diff
@@ -41,4 +41,43 @@ public void testCompareStringent(final String expectedFile, final String actualF
         final File actualF= new File(TEST_DATA_DIR, actualFile);
         Assert.assertEquals(expectedEqual, null == SamAssertionUtils.samsEqualStringent(actualF, expectedF, ValidationStringency.LENIENT, null));
     }
+
+    // make sure CRAM file assertions rejects non-CRAM files
+
+    @Test
+    public void testIsCRAMFileOnCRAM() {
+        Assert.assertTrue(SamAssertionUtils.isCRAMFile(
+                new File(TEST_DATA_DIR, "file1.cram"), new File(b37_reference_20_21)));
+    }
+
+    @Test
+    public void testIsCRAMFileOnBAM() {
+        Assert.assertFalse(SamAssertionUtils.isCRAMFile(
+                new File(TEST_DATA_DIR, "file1.bam"), new File(b37_reference_20_21)));
+    }
+
+    @Test
+    public void testAssertsIfCRAMFileCRAM() {
+        SamAssertionUtils.assertValidateContentsIfCRAM(
+                new File(TEST_DATA_DIR, "file1.cram"), new File(b37_reference_20_21));
+    }
+
+    @Test
+    public void testAssertIfCRAMFileBAM() {
+        SamAssertionUtils.assertValidateContentsIfCRAM(
+                new File(TEST_DATA_DIR, "file1.bam"), new File(b37_reference_20_21));
+    }
+
+    @Test
+    public void testAssertCRAMFileContentsCRAM() {
+        SamAssertionUtils.assertValidateContentsIfCRAM(
+                new File(TEST_DATA_DIR, "file1.cram"), new File(b37_reference_20_21));
```

Comment:
Use a `DataProvider` for these `assertValidateContentsIfCRAM()` tests as well, and include a test case with a fake cram and an expected exception.